
<launch>
    <node pkg="createmap" type="IROS_map" name="IROS" output="screen">
        <param name="config_file" type="string" value="home/zzh/maplab_src/src/createmap/config/test.yaml"/>
        <param name="is_fisheye" type="bool" value="true"/>
        <param name="camera_topic" type="string" value="/t265/fisheye1/image_raw "/>
        <param name="acc_topic" type="string" value="/t265/accel/sample"/>
        <param name="gyr_topic" type="string" value="/t265/gyro/sample"/>
        <param name="gt_topic" type="string" value="/gt"/>
        <param name="map_folder" type="string" value=""/>
    </node>
</launch>
